### DISCUSSION TITLE
Add a row for numbers

### DISCUSSION THREAD

**Discussion by luziato** [8VIM/8VIM#282] [Ideas]
Hi I'm new to this keyboard and I'm realy loving it.
What about a row of number on top of the keyboard? Like gboard and others. And you can select a number by swiping from center to that number (or just pressing)... for example: when you whant to input a capital letter in G-board you can press and hold then swipe from the "shift" key to the desired letter on the keyboard, when you do so a straight line appears and when you are done you just release and the capital letter are printed

For now i think a simple number row would do the job just fine and speed up the process a lot

Something like this 
![2023-2-9_15-16-38](https://user-images.githubusercontent.com/33085643/217837937-487a4453-57eb-4427-ac84-c333f2cc40a8.jpg)

I would like to fork to implement this feature but I don't have the right knowledge to do it, maybe someone can help me?

**Comment by flide** [comment]
Lol, that is the simplest and most sensible solution to the numbers problem
suggested till date.
